nitrite_test.go

```diff
@@ -0,0 +1,75 @@
+package nitrite_test
+
+import (
+	"crypto/x509"
+	_ "embed"
+	"encoding/base64"
+	"testing"
+	"time"
+
+	"github.com/blocky/nitrite"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+//go:embed testdata/nitro_attestation.txt
+var nitroAttestString string
+
+func TestNitrite_Verify(t *testing.T) {
+	attestBytes, err := base64.StdEncoding.DecodeString(nitroAttestString)
+	require.NoError(t, err)
+
+	doc := nitrite.Document{}
+	err = doc.FromCosePayloadBytes(attestBytes)
+	require.NoError(t, err)
+
+	roots := x509.NewCertPool()
+	ok := roots.AppendCertsFromPEM(nitrite.AWSNitroEnclavesCertPEM)
+	require.True(t, ok)
+
+	t.Run("happy path", func(t *testing.T) {
+		opts := nitrite.VerifyOptions{
+			CurrentTime: doc.CreatedAt(),
+			Roots:       roots,
+		}
+
+		result, err := nitrite.Verify(attestBytes, opts)
+		require.NoError(t, err)
+		assert.Equal(t, *result.Document, doc)
+		assert.True(t, result.SignatureOK)
+	})
+
+	t.Run("expired attestation", func(t *testing.T) {
+		opts := nitrite.VerifyOptions{
+			CurrentTime: time.Now(),
```

Comment:
Good point. Really what this is testing is that given a time outside of the validity window the vert verification should fail because it is either "expired or not yet valid". 

So, I've updated the test to use `time.UnixMilli(0)` (i.e., Jan 1 1970) as the current time instead of `time.Now()`. While there is still the assumption that the test attestation was not valid at jan 1 1970, that assumption is less problematic than assuming the attestation is not currently valid.